### ISSUE TITLE
Album.items() should sort by track number first

### ISSUE THREAD

**Issue by sampsyo** [sampsyo/beets#1497]
As pointed out by @djl in #494, the `Album.items()` method is returning tracks sorted by the global sort specification for items (the `sort_item` config option). By default, this is `artist+ album+ disc+ track+`, but this means that albums with multiple track artists appear "out of order" (since `artist` precedes `track`).

An easy way to fix this would be to make the default sort for items use `albumartist`, and not `artist`. This would have the nice effect of grouping albums correctly when searching for items—something that I believe that we had in the past. But it makes less sense for singletons, where album artists may not even be present.

Any thoughts?

**Comment by djl** [comment]
What about adding `FieldSort('track', case_insensitive=False)` to the [`Album.items()`](https://github.com/sampsyo/beets/blob/bc6f5869b4b1ec1b538c13e3b3db17d83f8aeacb/beets/library.py#L931) method? That should sort the tracks correctly for an album and not affect singletons.

**Comment by sampsyo** [comment]
You're right, that would be the alternative. (We should probably also include `disc`.)

But it would actually be nice, in my opinion, if `beet ls` by default grouped items by their album and sorted them in a natural way. But maybe that doesn't matter as much.

**Comment by Sjlver** [comment]
+1 for this. I tried `beet play -a` "[calling all dawns](https://musicbrainz.org/release/011ae209-eb8d-33c2-8d87-7999f1a97ab0)" today, and was quite confused because the tracks were out of order.

I've now set `sort_item: albumartist+ album+ disc+ track+` in my config, and think that it should be the default.
